bloom-housing/bloom
# 1430/listings management, fcfs and lottery

# Pull Request Template

## Issue

Addresses #1430 

## Description

Adds the lottery and first come first serve radio button selection to the rankings and results portion of listings management.

If you select FCFS, an additional radio question `Is there an application due date?` will appear - if you select `No`, the application due date field at the bottom of the page should become disabled

If you select Lottery, three additional fields for the date, start time, and end time, as well as a textbox field for notes should appear

I added a computed backend field to be able to tell if a listing is FCFS.

## Type of change


## How Can This Be Tested/Reviewed?

Add a new listing with FCFS and no due date, and check that the public listing displays "First come first serve"
Add a new listing with Lottery and check out the additional lottery section with the time range and notes in the right hand panel

## Checklist:
